libs/store/src/lib/loggers/player.ts

```diff
@@ -0,0 +1,36 @@
+import { AppErrorCode, AppMessageLogType } from '@razor/models';
+import { Dispatch } from '../store';
+
+export const playerNotFound = async (
```

Comment:
why do we have async functions here?
If `dispatch.games.sendLogMessage` is asynchronous, we need to `await` on the response.

Reply:
Fixed in PR #14 under commit 563c7da7291336f0b666f5c471e3d5046054c636